src/client/components/componentWithSliders/modalSavePressetesWindow/savePressetModalBlock.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component, Fragment } from 'react';
+import ModalSavePressetesWindow from './modalSavePressetesWindow';
+import BackgroundUnderModelWindow from './backgroundUnderModelWindow';
+
+class SavePressetModalBlock extends Component {
+    state = {
+      isModalBlockShow: false,
+    }
+
+    componentDidMount() {
+      const buttonSave = document.getElementById('savePressetes');
```

Comment:
No need to use don selectors while using react, try to change all in this method by using react

Reply:
rewrite using "ref"